disco_aws_automation/disco_vpc.py

```diff
@@ -537,6 +538,8 @@ def _destroy_instances(self):
         """ Find all instances in vpc and terminate them """
         autoscale = DiscoAutoscale(environment_name=self.environment_name)
         autoscale.clean_groups(force=True)
+        elastigroup = DiscoElastigroup(environment_name=self.environment_name)
+        elastigroup.delete_groups()
```

Comment:
Maybe we should make this method name consistent with the `DiscoAutoscale` version. Could rename the `DiscoAutoscale` one to `delete_groups` to match `DiscoElastiGroup`